src/modules/dashboard/components/TransactionDataTable/FailedTxnDT.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+    <div>
+      <DataTable :value="transactions" :paginator="true" :rows="10" scrollDirection="horizontal"
+        :alwaysShowPaginator="false" responsiveLayout="scroll"
+        paginatorTemplate="CurrentPageReport FirstPageLink PrevPageLink PageLinks NextPageLink LastPageLink RowsPerPageDropdown"
+        currentPageReportTemplate="">
+        <Column style="width: 200px" headerClass="invisible" v-if="!wideScreen">
+          <template #body="{ data }">
+            <div>
+              <div class="uppercase text-xxs font-bold mb-1">{{ $t('dashboard.txHash') }}</div>
+              <div class="flex items-center">
+                <div class="uppercase cursor-pointer font-bold text-txs">
+                    <span class="text-txs" v-tooltip.right="data.hash">{{ data.hash.substring(0, 20) }}...</span></div>
+              </div>
+            </div>
+          </template>
+        </Column>
+        <Column style="width: 200px" headerClass="invisible" v-if="!wideScreen">
+          <template #body="{ data }">
+            <div>
+              <div class="uppercase text-xxs font-bold mb-1"> Status </div>
+              <div class="flex items-center">
+                <span class="text-txs">{{ data.status }}</span>
+              </div>
+            </div>
+          </template>
+        </Column>
+        <Column field="hash" :header="$t('dashboard.txHash')" headerStyle="width:100px" v-if="wideScreen">
+          <template #body="{ data }">
+            <div class="flex items-center">
+              <span class="text-txs cursor-pointer" v-tooltip.bottom="data.hash">{{ data.hash.substring(0, 20) }}...</span>
+            </div>
+          </template>
+        </Column>
+        <Column field="formattedDeadline" :header="$t('general.deadline')" headerStyle="width:110px" v-if="wideScreen">
+          <template #body="{ data }">
+            <span class="text-txs">{{ convertLocalTime(subtractHours(data.deadline.value, 2)) }}</span>
+          </template>
+        </Column>
```

Comment:
wrongly done, it is a deadline, there is no way to get the time when  announcing the txn. Anyway, it is not needed as it is meaningless to user